packages/application-to-register-webapp/src/utils/register-task-list.js

```diff
@@ -0,0 +1,63 @@
+import constants from './constants.js'
+
+const REGISTER_TASK_LIST = {
```

Comment:
One last thing, just realised. This object shouldn't be in capitalised snake case, as it is not a constant as such, but an object/class that is manipulated.  Please change to pascal, and all usage of it in code files.

Reply:
It is a constant and the code does not manipulate that object directly as such. Every time the code makes a copy of it, leaving it intact.

Reply:
Renamed to camelcase